src/wovnio/html/HtmlConverter.php

```diff
@@ -278,6 +279,18 @@ function _removeWovnIgnore($node, $marker) {
     }
   }
 
+  function _removeCustomIgnoreClass($node, $marker) {
+    $class_attr = $node->getAttribute('class');
+    if ($class_attr) {
+      $classes_to_ignore = $this->store->settings['ignore_class'];
+      $classes = array_filter(explode(' ', $class_attr));
```

Comment:
Do you think it is enough to split by space? Maybe we should split by whitespace?

Reply:
I made the changes, thanks.

Reply:
thanks